lambdas/handlers/bulk_upload_handler.py

```diff
@@ -13,35 +12,27 @@
 @override_error_check
 def lambda_handler(event, _context):
     logger.info("Received event. Starting bulk upload process")
+
+    http_status_code = 200
+    response_body = f"Finished processing all {len(event['Records'])} messages"
```

Comment:
Maybe could we move these two lines to between lines 29 and 30 for clarity?  So that it is easier to see which status code and response body are related to which scenario.